frb_codegen/src/commands.rs

```diff
@@ -274,3 +283,88 @@ pub fn build_runner(dart_root: &str) -> Result {
     }
     Ok(())
 }
+
+pub fn guess_context(dart_root: &str) -> std::result::Result<Context, Error> {
```

Comment:
uber-nit:

* maybe no need for `pub`？
* maybe use `anyhow::Result`?

Reply:
fixed in d93a792ef32befbcf28331b7fe6dca3be5630e9c and fa4ecba03052ef00fc754fc3636c7519735c54cf